cmd/batch_test.go

```diff
@@ -214,6 +220,68 @@ var _ = Describe("Batch", func() {
 		Expect(afs.WriteFile(headerPath, headerData, os.ModePerm)).To(Succeed())
 	}
 
+	It("can add new lines before kind header", func() {
```

Comment:
you can combine before and after new line tests into one to reduce the test count a little bit. Might make it a little easier to get all the tests we need done this way.